L1/BubbleSort.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+
+
+def data(start,stop,size):
+    lista = []
+    for i in range(int(size)):
+        nl = randint(start, stop)
+        lista.append(nl)
+    return lista
+
+
+def bubblesort(data):
+    nd = data[:]
+    for i in range(len(nd) - 1, 0, -1):
+        for j in range(i):
+            if nd[j] > nd[j + 1]:
+                nd[j], nd[j + 1] = nd[j + 1], nd[j]
+    return nd
+
+
+a = data(10, 100, 10)
```

Comment:
data nic nie mowi, jesli tworzysz funkcje pomocnicza generujaca dane z zakresu to tak ja nazwij. Jeszcze gorzej z jej argumentami start stop, powinny raczej sie nazywac tak zeby czlowiek sie domyslil ze to jest dol i gora zakresu.